dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/InvokeTelnetHandler.java

```diff
@@ -119,26 +120,30 @@ private static boolean isMatch(Class<?>[] types, List<Object> args, Class<?>[] p
     @SuppressWarnings("unchecked")
     public String telnet(Channel channel, String message) {
         if (message == null || message.length() == 0) {
```

Comment:
Can be change to 
`if(!StringUtils.isEmpty(message))`

Reply:
done.